### DISCUSSION TITLE
Failed to register

### DISCUSSION THREAD

**Discussion by joneswong** [snap-stanford/ogb#181] [PCQM4M-LSC]
Hi administrators,
    I encountered some web crash issue and thus submitted the sheets twice. However, I have not received the confirm email. Since I am not allowed to post in that google group, I have to ask for help here. Could you help me correcting the registration? Thanks!

**Comment by weihua916** [comment]
Hi! We saw your two identical registrations and deleted one of them. Please check your spam folder for the receipt. For the subsequent inquiry, please reach out to us at ogb-lsc@cs.stanford.edu since some of your registration info cannot be shared publicly.
